### DISCUSSION TITLE
DM8 国产数据据库不能用吧

### DISCUSSION THREAD

**Discussion by 512744006** [apache/incubator-seata#5733] [General]
DM8  达梦8 国产数据据库不能用，现在信创都在搞，这个不支持，感觉不好办。现在很多系统尤其关系国计民生的系统，系统、数据库都在转国产，如果seata,不支持感觉，可能用的人会慢慢少了

**Comment by funky-eyes** [comment]
欢迎你参与到达梦pr的review 测试 共建中

**Reply by kyle-zeng** [reply]
目前用1.8.0版本，达梦数据库，写库时报，#Cause: io.seata.common.loader.EnhancedServiceNotFoundException: not found service provider for : io.seata.sqlparser.druid.SQLOperateRecognizerHolder
能否帮忙排查下是啥原因呢？
[seata-dm.txt](https://github.com/seata/seata/files/13331572/seata-dm.txt)

**Reply by funky-eyes** [reply]
Check for dependency conflicts @kyle-zeng

**Reply by kyle-zeng** [reply]
是包引入问题，前一个版本的引用包没有删除导致，谢谢！

**Comment by slievrly** [comment]
will support in 1.8.0.

**Comment by leizhiyuan** [comment]
<img width="1265" alt="image" src="https://github.com/seata/seata/assets/2684384/33770918-e4fc-43b7-ab93-6d379726e746">


can you find this file in your project?

**Reply by leizhiyuan** [reply]
and please provide your config of db connection
